steps/compliance/dwp-issuing-office/DWPIssuingOffice.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { Question, goTo } = require('@hmcts/one-per-page');
+const { form, textField } = require('@hmcts/one-per-page/forms');
+const { numbers } = require('utils/regex');
+const officeIds = require('steps/compliance/dwp-issuing-office/dwpIssuingOfficeIds');
+const Joi = require('joi');
+const paths = require('paths');
+
+class DWPIssuingOffice extends Question {
+
+    static get path() {
+
+        return paths.compliance.dwpIssuingOffice;
+    }
+
+    get form() {
+
+        return form(
+
+            textField('pipNumber')
+                .joi(
```

Comment:
Sorry, only just seen this. Can you move the `.joi` up a line so we are consistent with all other steps. AppellantContactDetails is a good example.